packages/app/src/components/drawer/confirm-send-drawer.tsx

```diff
@@ -40,12 +40,17 @@ export const ConfirmSendDrawer: React.FC<ConfirmSendDrawerProps> = ({
   balances,
 }) => {
   const [tx, setTx] = useState<StacksTransaction | undefined>();
-  const { currentIdentity, currentNetwork, doSetLatestNonce } = useWallet();
+  const {
+    currentAccount,
+    currentNetwork,
+    doSetLatestNonce,
+    currentAccountStxAddress,
+  } = useWallet();
   const asset = useRecoilValue(selectedAssetStore);
   const [loading, setLoading] = useState(false);
   const { doChangeScreen } = useAnalytics();
   const getTx = useCallback(async () => {
-    if (!asset || !currentIdentity) return;
+    if (!asset || !currentAccount || !currentAccountStxAddress) return;
     const network = new StacksTestnet();
```

Comment:
should this be hardcoded to the testnet?

Reply:
I don't have the same ChainID logic that you have in my network store. Once I have that, `currentAccountStxAddress` will be a recoil selector.